resources/views/components/title/base.blade.php

```diff
@@ -1,14 +1,20 @@
 @props(['tag' => 'p'])
 @php
-    $isAnchor = $attributes->has('href', ':href', 'v-bind:href');
+    $isAnchor = count(
+        $attributes
+            ->only('href', ':href', 'v-bind:href')
+            ->filter(fn($value) => isset($value))
+            ->getAttributes(),
+    );
```

Comment:
Sick shizzle me and @indykoning made, this checks if the attribute has an href and if it actually has a value set. Not sure if this could be done better

Reply:
Filter without a callback will filter out `false` values, doesn't that cover it already? Maybe create a PR to Laravel so `has()` supports multiple values?

Everything can be minimised to something like this:
```
@props(['tag' => count($attributes->only('href', ':href', 'v-bind:href')->filter(null)->getAttributes()) ? 'a' : 'p'])
```
or if you think that's to long:
```
@props(['tag' => count($attributes
    ->only('href', ':href', 'v-bind:href')
    ->filter(null)
    ->getAttributes())
        ? 'a' 
        : 'p'
])
```